### DISCUSSION TITLE
Add Mantic to https://aquasecurity.github.io/trivy-repo/deb/

### DISCUSSION THREAD

**Discussion by ilar** [aquasecurity/trivy#6570] [Ideas]
### Description

There are no Mantic releases for Trivy in the Debian/Ubuntu repo.

### Target

None

### Scanner

None

**Comment by DmitriyLewen** [comment]
Hello @ilar 

We added `mantic` for Trivy `v0.51.1` - https://github.com/aquasecurity/trivy-repo/tree/main/deb/dists/mantic

Regards, Dmitriy

**Comment by ilar** [comment]
Thanks!
